core/src/Streamly/Data/Unfold.hs

```diff
@@ -226,3 +230,58 @@ import Streamly.Internal.Data.Unfold
 -- streams are written, it is easy to adapt a stream to an unfold. If you are
 -- writing an unfold you can convert it to stream for free using
 -- 'Stream.unfold'.
+
+-- | Unfolds @from@ generating a stream starting with the element
```

Comment:
The API is called `enumerate`
The documentation needs to be changed!
Why is it named `enumerate` and not `enumerateFrom`?
Why not change the name to `enumerate` in the internal module itself?

Reply:
The name "enumerate" API is used in Stream, so to be the matching name.